src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/HotReload/Contracts/IProjectHotReloadAgent.cs

```diff
@@ -1,28 +1,28 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
+using System.Runtime.CompilerServices;
 using Microsoft.VisualStudio.ProjectSystem.Debug;
-using Microsoft.VisualStudio.ProjectSystem.HotReload;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.HotReload;
 
+[InternalImplementationOnly]
```

Comment:
This is not currently verified, but it serves as marker that we don't expect the interface to be implemented outside of this assembly.